lockbox-ios/Common/Helpers/KeyManager.swift

```diff
@@ -18,15 +18,17 @@ class KeyManager {
         return String(cString: jsonValue!)
     }
 
-    func decryptJWE(_ jwe: String) -> String {
+    func decryptJWE(_ jwe: String) -> String? {
         let count = jwe.data(using: .utf8)!.count as size_t
         let contentLen = UnsafeMutablePointer<size_t>.allocate(capacity: count)
         let err = UnsafeMutablePointer<cjose_err>.allocate(capacity: count)
 
         let cJoseJWE = cjose_jwe_import(jwe, count, nil)
-        let decryptedPayload = cjose_jwe_decrypt(cJoseJWE, self.jwk, contentLen, err)
+        guard let decryptedPayload = cjose_jwe_decrypt(cJoseJWE, self.jwk, contentLen, err) else {
+            return nil
```

Comment:
The only reason `cjose_jwe_decrypt()` returns `NULL` is if it fails.  This might want to "throw" `err` in some form, or at least log it somehow.

## background ##
[`cjose_err`](http://cisco.github.io/cjose/structcjose__err.html) is a structure with a bunch of static values in it, but the most relevant for logs (if nothing else) are:

* `code` describing the error in a machine-readable way
* `message` canned text describing the error for a human (more or less)